fixtures/sdk_crash_detection/crash_event_react_native.py

```diff
@@ -0,0 +1,172 @@
+from typing import Any, Collection, Dict, Mapping, MutableMapping, Sequence
+
+
+def get_frames(filename: str) -> Sequence[MutableMapping[str, Any]]:
```

Comment:
Any reason to not map str -> str?
```suggestion
def get_frames(filename: str) -> Sequence[MutableMapping[str, str]]:
```